TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -253,18 +267,51 @@ public async Task<CommandResult> Pokedex(CommandContext context)
                     ResponseTarget = ResponseTarget.WhisperIfLong
                 };
             }
+            else if (mode.Equals(PokedexModeKantoooooooo))
+            {
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == Generation.Gen1);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                };
+            }
+            else if (_pokedexModeRegions.ContainsKey(mode.ToLower()))
+            {
+                Generation modeGeneration = _pokedexModeRegions[mode.ToLower()];
+
+                // Getting the total number of distinct Pokemon from a single generation could also be a function in class PokedexData,
+                //   by using "GenMaxIds", but I'm not sure how to get the total number of Fakemons
+                int totalRegionCount = _knownSpecies.Count(pokemon => pokemon.GetGeneration() == modeGeneration);
```

Comment:
This could be done once in the constructor for all generations instead of each time the command runs